### PR TITLE
allow tracing for non-lambda servers

### LINKED ISSUE #36: Proxy: tracing doesn't work
There is this error in AE:

> This trace information is incomplete. Check the product documentation to properly enable tracing in your GraphQL server and ensure you the latest version of the Engine Proxy is used.

Tracing doesn't work (only timing) and as a consequence, we cannot see error details:

screen shot 2018-03-14 at 16 51 58

I am now in the situation where one client fails but I don't know why and where to start.

### PR DESCRIPTION
**Pull request by Fallion**
Closes #36

### CODE REVIEW

**Comment by Fallion** [comment]
screen shot 2018-03-19 at 19 42 50

File: src/graphqlServer.js [hunk 1]
```diff
@@ -67,8 +67,7 @@ function createGraphqlServer(schema, context) {
     },
     extensions: () => {
       const traceCollector = context._traceCollector;
-      if (true || !traceCollector) return {}; // eslint-disable-line no-constant-condition
-      // Quick hack, remove 'true ||' once we can handle huge tracing output
+      if (!!process.env.IS_LAMBDA || !traceCollector) return {};
```
Comment (michalsanger) [inline src/graphqlServer.js]: Isn't it better not to setup the tracing on Lambda at all? So put the `process.env.IS_LAMBDA` check on line 37

Comment (Fallion) [reply src/graphqlServer.js]: Good point. Didn't see that, will update now.

Comment (Fallion) [reply src/graphqlServer.js]: Fixed.

**Pull request merged**